_posts/2021-01-14-voicemeeter-banana.md

```diff
@@ -0,0 +1,112 @@
+
+Location of Virtual Audio Cable in Voicemeeter after installation:
+'C:\Program Files (x86)\VB\Voicemeeter'
+
+
+## Setting up Voicemeeter Banana
+
+Please open Voicemeeter Banana.
+
+### Saving Settings, Loading Settings, Resetting settings
+Something that typically gets skipped in other Voicemeeter coverage that i've seen is the ability to save settings. At any point you can save your current settings of voicemeeter by going to Menu→Save Settings. This saves a settings file on your computer you can then load up from just a file even on other devices should you want to transfer your settings to another device. If at any point you feel lost and want to restart the setup, you can go to Menu→ Reset Settings (Re-Initialization) and this will make Voicemeeter appear as if you had just installed it, blank with nothing set up.
+
+Image for showing where the menu is
+
+![https://i.imgur.com/bbE1y24.png](https://i.imgur.com/bbE1y24.png)
+
+Image for showing save/load/reset settings
+
+![https://i.imgur.com/bbE1y24.png](https://i.imgur.com/bbE1y24.png)
+
+
+### Audio outputs
+The Hardware Ouputs section in the top right should have A1, A2, A3 with a triangle beside them. Each one of these have a selectable menu for our output devices. Each input device has a toggle-able setting for us to push audio toe each individual output device, which will get  better explanation when we get to inputs inputs. For now, click on A1 and select your headphones. click on A2 and select the virtual audio cable.
+
+Your A1, A2, and A3 section should look a little something like this:
+
+![https://i.imgur.com/Y5BvneM.png](https://i.imgur.com/Y5BvneM.png)
+
+Below that you will see a bunch of vertical bars, for now we will leave these alone until we have audio pushing through them. For that, we will start by pushing computer audio to them through a digital inputs.
+
+
+### Digital Inputs and Windows Audio
+For digital inputs we will be looking at the middle columns of Voicemeeter Banana which should look a little something like this:
+
+Voicemeeter Digital Inputs:
+
+![https://i.imgur.com/pkMT14v.png](https://i.imgur.com/pkMT14v.png)
+
+To set our Windows Default Audio to push through these digital inputs we need to open Windows Sound Settings by right clicking on the speaker icon that adjusts our audio in the task bar. After right clicking select 'Open Sound Settings' and you should see a window with
+
+Sound-
+Choose your output device:
+
+![https://i.imgur.com/0XNcvOK.png](https://i.imgur.com/0XNcvOK.png)
+
+Select VoiceMeeter Input (VB-Audio Voicemeeter VAIO) as the default
+
+![https://i.imgur.com/0XNcvOK.png](https://i.imgur.com/0XNcvOK.png)
+
+After setting VB-Audio Voicemeeter VAIO as our default output device pull up Voicemeeter Banana. In the middle under the Voicemeeter VAIO section we should see the audio levels rising and lowering if you play sounds on our computer. Do you? Great! If not? Try again!
```

Comment:
May add something in here about popping into the Discord server for more help if they need it.

Reply:
added this in.